javascript-bank/account.js

```diff
@@ -0,0 +1,56 @@
+/* exported Account */
+function Account(number, holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+
+}
+
+Account.prototype.deposit = function (amount) {
+  this.amount = amount;
+  this.newDeposit = [];
+  this.success = false;
```

Comment:
Please do not add additional properties not requested in the instructions. You can make local variables to the function to get the same effect without polluting the object the data is being stored in.